src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim()) {
+      const [property, value] = declaration.split(':');
+
+      styleObject[property.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Consider using `.map()` to transform each declaration into a key-value pair, and then use `.filter()` to eliminate any empty declarations before adding them to the `styleObject`.